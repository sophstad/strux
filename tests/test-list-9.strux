int main() {
	LinkedList::string l = new LinkedList::string("hello", "world");
	string s = l.get(0);
    print(s);

    LinkedList::num l2 = new LinkedList::num(3.0, 4.0, 5.0);
    num n = l2.get(1);
    print(n);
    
    LinkedList::int l3 = new LinkedList::int(100, 1000, 10000);
    int n2 = l3.get(1);
    print(n2);

    LinkedList::bool l4 = new LinkedList::bool(true, false, true, false, true);
    bool b = l4.get(3);
    print(b);

	return 0;
}
