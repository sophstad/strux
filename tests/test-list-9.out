hello
4.000000
1000
0
